rotkehlchen/chain/evm/tokens.py

```diff
@@ -487,7 +493,12 @@ def query_tokens_for_addresses(
                         continue  # skip any zero balance tokens
 
                     tokens_with_balance.add(token)
-                    addresses_to_balances[address][token] = balance
+                    if token.token_kind == TokenKind.ERC721:
+                        # For nfts the balance will be the total number of nfts the user owns from
```

Comment:
it brings to my mind some kind of similar bug that we had in other place but I can't remember where